### PR TITLE
docs(logo): DLT-1866 modify dialtone documentation logo

### PR DESCRIPTION
**Pull request by hynes-dialpad**
# Modify Dialtone's documentation logo

## Obligatory GIF (super important!)
batman-all-the-logos-all-the-time

- :hammer_and_wrench: Change type: Documentation
- :book: Jira Ticket: [DLT-1866](https://dialpad.atlassian.net/browse/DLT-1866)

## :book: Description

Instead of showing the "Dialpad" logo for the "Dialtone" website, this change modifies the logo area to be:

```
[ DP glyph ] + Dialtone
```
It will look like this:
image

[DLT-1866]: https://dialpad.atlassian.net/browse/DLT-1866?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
     title="Go back to the homepage"
     to="/"
   >
-    <svg-loader :illustration="true" name="dialpad-logo" />
+    <dt-stack gap="400" direction="row">
+      <dt-icon
+        name="dialpad-ai-color-reversed"
+        size="500"
+      />
+      <span class="d-fs-300 d-fw-bold">Dialtone</span>
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue:12]: Bold feels far too overpowering. Suggest semibold, maybe even medium.

```suggestion
      <span class="d-fs-300 d-fw-semibold">Dialtone</span>
```

File: .nx/workspace-data/d/daemon.log [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Error: Cannot find module '/Users/hynes/Projects/dialtone/node_modules/.pnpm/nx@19.2.3/node_modules/nx/src/daemon/server/start.js'
+    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
+    at Module._load (node:internal/modules/cjs/loader:975:27)
+    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
+    at node:internal/main/run_main_module:28:49 {
+  code: 'MODULE_NOT_FOUND',
+  requireStack: []
+}
+
+Node.js v18.19.0
```
Comment (francisrupert) [inline .nx/workspace-data/d/daemon.log]: Note sure why this got committed, let alone created. I'd have expected it to be ignored at least. Probably can delete it.

File: apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue [hunk 3]
```diff
@@ -1,14 +1,15 @@
 <template>
   <router-link
-    class="d-btn h:d-bgc-transparent d-py6 d-w216 d-jc-flex-start"
+    class="d-btn d-btn--muted h:d-bgc-transparent d-py6 d-px12 d-w216 d-jc-flex-start"
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue]: Probably shouldn't have been a `d-btn` to begin with. 
```suggestion
    class="d-pl8 d-w216 d-fc-secondary d-td-none"
```

**Review by francisrupert — COMMENTED** [review]
OK cool. Like. A few comments and suggestions...

File: .nx/workspace-data/d/daemon.log (hunk 2 above)
Comment (braddialpad) [reply .nx/workspace-data/d/daemon.log]: interesting.. that entire folder should be gitignored. Yeah you can delete

**Comment by braddialpad** [comment]
Looks like something is wrong with the lock file... let me see if I can fix that

**Comment by braddialpad** [comment]
Alright you're good.

**Review by braddialpad — APPROVED** [review]
Looks good, can merge after making francis' requested fixes.

**Comment by iropolo** [comment]
huh there is a pnpm-lock file in the PR, should we remove it? @braddialpad

**Comment by braddialpad** [comment]
> huh there is a pnpm-lock file in the PR, should it go? @braddialpad

I just ran pnpm install and that's what it did. Should be fine I guess 🤷 . Maybe staging lockfile was out of date

**Review by iropolo — APPROVED** [review]

File: apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue (hunk 1 above)
Comment (hynes-dialpad) [reply apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue:12]: Let's leave it for now. We can always reduce it in the future. 😄

**Review by francisrupert — APPROVED** [review]
🚀

**Pull request merged**